v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/options/KafkaToBigQueryFlexOptions.java

```diff
@@ -51,8 +51,12 @@ public interface KafkaToBigQueryFlexOptions
         @TemplateParameter.TemplateEnumOption(KafkaAuthenticationMethod.SASL_PLAIN),
         @TemplateParameter.TemplateEnumOption(KafkaAuthenticationMethod.NONE),
       },
-      description = "Authentication Mode",
-      helpText = "Kafka read authentication mode. Can be NONE or SASL_PLAIN")
+      description = "Kafka Read Authentication Mode",
+      helpText =
+          "The mode of authentication to use with the Kafka cluster. "
+              + "Use `NONE` for no authentication, `SASL_PLAIN` for SASL/PLAIN username and password, "
+              + "SASL_PLAIN for SASL/PLAIN username/password, "
```

Comment:
I think this line should be TLS instead of SASL_PLAIN, or if TLS isn't an option, we shouldn't repeat SASL_PLAIN.

Reply:
Removed that duplicate line.